pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,121 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc {
+    //index-client
+    rpc BatchWrite(WriteBatch) returns (Nothing);
+    rpc QueryPages(IndexQuery) returns (ReadBatch);
```

Comment:
We shouldn't put the Cassandra limitation of a single query at once into the grpc layer. Should be `rpc QueryPages([]IndexQuery) returns (ReadBatch);`, which should return a streaming response, with each read on the stream returning a `ReadBatch`.

Reply:
@gouthamve Made required changes to support multiple rows per stream.